src/util/server.c

```diff
@@ -518,12 +507,18 @@ int server_setup(const char *name, int flags,
     }
 
     if (flags & FLAGS_PID_FILE) {
```

Comment:
If I see it correctly `FLAGS_PID_FILE` is only used in a test, so the flag and the full if-block can be removed. As a result `get_pid_path()` has no callers anymore and can be removed as well and the defined constants from `util.h` can be moved to `monitor.c`.

Reply:
> If I see it correctly FLAGS_PID_FILE is only used in a test,

Yes, it is.

> so the flag and the full if-block can be removed

Why? As far as I understand, this would break `tests/cwrap/test_server.c` as `wait_for_bg_server()` expects pidfile to be created by this block.

Reply:
ah, I'm sorry, you are right. I just saw that it is used in a test and assumed only the functionality was tested, I didn't realize that the pid file was needed by the test. In this case the block must of course be kept.

Reply:
Ok, thanks, I guess I may mark this as resolved.
Is anything left?